app/src/main/res/values/colors.xml

```diff
@@ -66,9 +66,9 @@
     <!-- Small heading, Text link -->
     <color name="fx_mobile_text_color_accent_disabled" tools:ignore="UnusedResources">@color/photonViolet70A80</color>
     <!-- Text Inverted/On Color -->
-    <color name="fx_mobile_text_color_oncolor_primary" tools:ignore="UnusedResources">@color/photonLightGrey05</color>
+    <color name="fx_mobile_text_color_oncolor_primary">@color/photonLightGrey05</color>
     <!-- Text Inverted/On Color -->
-    <color name="fx_mobile_text_color_oncolor_secondary" tools:ignore="UnusedResources">@color/photonLightGrey40</color>
+    <color name="fx_mobile_text_color_oncolor_secondary">@color/photonLightGrey40</color>
```

Comment:
with these longer color names, do we define anywhere what the terms "oncolor, primary, secondary, etc." mean?

Reply:
We would ideally want https://www.figma.com/file/YcQC7rU1SQJx49kIAXZhcI/Component-and-Color-Usage-Examples?node-id=98%3A16590 updated with examples.

You can refer to https://github.com/mozilla-mobile/fenix/pull/23038#issuecomment-1020289848. 
Primary is for the primary text (main/body/title text). Secondary is for caption/subtitle/subtext. Oncolor basically means the text is in a dark background so the text color needs to be inverted from black to white.